docs/development.rst

```diff
@@ -0,0 +1,57 @@
+.. highlights:: shell
+
+===============
+Developer guide
+===============
+
+
+Recommended development environment setup
+-----------------------------------------
+
+To setup development environment, run this command in your terminal:
+
+.. code-block:: console
+
+    $ conda env create -f=environment.yml
```

Comment:
If someone runs this command, it will not work. There are two reasons for this:
1. There is no equals sign (=) in the conda command, and
2. The file(s) we need for the dask-image test environments are not named `environment.yml`